src/rpc/rawtransaction.cpp

```diff
@@ -825,7 +825,7 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
 
     RPCTypeCheck(request.params, {
         UniValue::VSTR,
-        UniValueType(), // NUM or BOOL, checked later
+        UniValueType(), // VNUM or VSTR, checked inside AmountFromValue()
```

Comment:
The previous comment said "NUM of BOOL" but it would actually accept "NUM or BOOL or STR" - we allowed BOOL explicitly and if not BOOL, then passed it to `AmountFromValue()` which would accept NUM or STR and throw otherwise.

Reply:
side-note: NUM or STR is often just referred to as AMOUNT